src/components/shared/ContextMenu.css

```diff
@@ -123,13 +123,17 @@
 /* Use this wrapper for a content that needs to take the available space or if
  * it contains other HTML elements (eg <strong>) */
 .react-contextmenu-item-content {
-  flex: auto;
+  overflow: hidden;
+  min-width: 0;
```

Comment:
super nit: this `min-width` seems redundant.
Indeed: `flex: 1` means `flex: 1 1 0`, which means `flex-basis: 0`. This is actually the main difference between `flex: auto` and `flex: 1` : -) I feel like `min-width` is `0` implicitely thanks to either the ellipsis or this `flex-basis: 0`.

Actually, here I think that `flex: auto` works too (thanks for the 0 shrink for the icon). But I don't mind which one we choose as both work.

Reply:
With the source view tab bar I ran into a case where `flex-basis: 0` was not enough to make the element shrink, I also needed `min-width: 0`: https://codepen.io/mstange/pen/XWgjJwK